File: CS50_2017/Lecture1/Problemset1/mario.c
#include <stdio.h>
#include <cs50.h>

void makePyramid(int n);
void makeSpace(int n);
void makeBlock(int n);

int main(void)
{
    int height = -1;
    while (height < 0 || height > 23)
    {
        printf("Height :");
        height = get_int();
    }
    makePyramid(height);
}

void makePyramid(int n)
{
    for (int i = 0; i < n; i++)
    {
        makeSpace(n - (i + 1));
        makeBlock(i + 2);
        printf("\n");
    }
}

void makeSpace(int n)
{
    for (int i = 0 ; i < n; i++)
    {
        printf(" ");
    }
}

void makeBlock(int n)
{
    
    for (int i = 0 ; i < n; i++)
    {
        printf("#");
    }
}

File: CS50_2017/Lecture8/Problemset6/mario.py
import sys
import cs50

def main():
    height = -1
    while height < 0 or height > 23:
        print("Height : ", end = "")
        height = cs50.get_int()
    makePyramid(height)

def makePyramid(n):
    for i in range(n):
        makeSpace(n - (i + 1))
        makeBlock(i + 2)
        print("")

def makeSpace(n):
    for i in range(n):
        print(" ", end = "")

def makeBlock(n):
    for i in range(n):
        print("#", end = "")
        
if __name__ == "__main__":
    main()
    